src/server/worker/stats.go

```diff
@@ -0,0 +1,115 @@
+package worker
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// PrometheusPort is the port the aggregated metrics are served on for scraping
+	PrometheusPort = int32(9090)
+
+	datumCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pachyderm",
+			Subsystem: "user",
+			Name:      "datum_count",
+			Help:      "Number of datums counted by pipeline and state",
```

Comment:
Small nit, but I think "Number of datums processed by pipeline" is a bit more accurate. I'm not quite sure what the "and state" at the end means.